airflow/upgrade/rules/custom_operator_metaclass_rule.py

```diff
@@ -17,6 +17,7 @@
 
 from __future__ import absolute_import
 
+from airflow import conf
```

Comment:
```suggestion
from airflow.configuration import conf
```

Reply:
What is the difference?
This is an example of rule that use it as I did:
https://github.com/apache/airflow/blob/v1-10-stable/airflow/upgrade/rules/import_changes.py#L24

This is an example of rule that use it as you suggest:
https://github.com/apache/airflow/blob/v1-10-stable/airflow/upgrade/rules/hostname_callable_rule.py#L20

Reply:
Hi @kenfinster, I understand there is such usage (the example you have nicely given). But I think it's better to adopt @ephraimbuddy 's suggestion even if both styles work.

One of the reasons can be to promote the "Airflow 2.0 style". `from airflow import conf` doesn't work in 2.0.x.

Reply:
Hi, @kenfinster, it's just a style difference, I'm Ok with it if you don't want to change it